docs/examples/exporters/GLTFExporter.html

```diff
@@ -47,33 +47,34 @@ <h3>[name]( )</h3>
 
 		<h2>Methods</h2>
 
-		<h3>[method:null parse]( [param:Object input], [param:Function onCompleted], [param:Object options] )</h3>
+		<h3>[method:null parse]( [param:Object3D input], [param:Function onCompleted], [param:Object options] )</h3>
 		<div>
 		[page:Object input] — Scenes or objects to export. Valid options:<br />
 		<ul>
 			<li>
 				Export scenes
 				<code>
-					exporter.parse( scene1, ...)
-exporter.parse( [ scene1, scene2 ], ...)
+				exporter.parse( scene1, ... )
+		exporter.parse( [ scene1, scene2 ], ... )
```

Comment:
This non-aligned indent is on purpose here and below to display aligned indent.
Browser's issue of code tag in li tag?